### ISSUE TITLE
Improve accessibility

### ISSUE THREAD

**Issue by mezeitamas** [mezeitamas/brokenrobot.xyz#1]
It is important to make a website accessible because everyone, regardless of their abilities or disabilities, should have equal access to the information and services provided by the website. So we need to make sure that everyone can navigate and interact with websites using assistive technologies, such as screen readers or keyboard navigation. Accessibility also benefits everyone, becaue it can provide an alternative way to access content.

## Useful links
- https://www.w3.org/WAI/fundamentals/accessibility-intro/
- https://www.a11yproject.com/
- https://web.dev/accessible/
- ESLint
  - https://github.com/evcohen/eslint-plugin-jsx-a11y
- React
  - https://legacy.reactjs.org/docs/accessibility.html
  - https://react-spectrum.adobe.com/react-aria/
- Gatsby
  - https://www.gatsbyjs.com/docs/conceptual/making-your-site-accessible/
- Testing
  - https://www.24a11y.com/2017/writing-automated-tests-accessibility/
